.github/workflows/ci-docs.yml

```diff
@@ -1,4 +1,8 @@
 name: Documentation
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: |-
+    ${{ github.ref != format('refs/heads/{0}', github.event.repository.default_branch) }}
```

Comment:
@shenvitor I smuggled in this DX improvement: it aborts CI if additional commits are pushed, in order to save CI time 🌍